server/middlewares/validations/ArticleRatingValidations.js

```diff
@@ -0,0 +1,39 @@
+import { validationResult, checkSchema } from 'express-validator/check';
+
+const handleValidationErrors = (req, res, next) => {
```

Comment:
This is called multiple times in several files. Consider making a helper function to aid re-usability?

Reply:
sorted